Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -627,10 +510,10 @@ public function isClassAbstract($className)
      * Tells if the specified class is final or not
      *
      * @param string $className Name of the class to analyze
-     * @return boolean true if the class is final, otherwise false
+     * @return bool true if the class is final, otherwise false
```

Comment:
This could be left out, as the comment does not add information that's not already in the name…

Reply:
> This could be left out, as the comment does not add information that's not already in the name…

well, yes … but I guess the ReflectionService will change a lot anyway during the next days …